tests/test_is_output_correct.py

```diff
@@ -24,9 +24,15 @@
         (5, 5.0),
         ("1/2", 0.5),
         ("\\frac{1}{2}", 0.5),
+        ("\\frac {1}{2}", 0.5),
         ("x^2+2x+1", "x^2 + 2*x + 1"),
         ("x^2+2x+1", "x^2 + 2*x - (-1)"),
         ("x^2+2x+1", "2x+ 1+x^2"),
+        ("17\\text{ any text}", "17"),
+        ("\$10", "10"),
+        ("10%", "1/10"),
```

Comment:
I think this is a bit hard to support in our current implementation. So let's maybe ignore errors like that. It will work if you replace 1/10 with 0.1